cookbooks/travis_build_environment/recipes/virtualenv.rb

```diff
@@ -0,0 +1,8 @@
+
+
+include_recipe 'travis_build_environment::pip'
+
+execute 'pip install virtualenv==15.1.0' do
+  version '15.1.0'
+  action :install
+end
```

Comment:
These three lines inside the block are not valid for the `execute` resource, so they can be removed.